Resources/Prototypes/_Impstation/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing, BaseCommandContraband]
+  id: ClothingOuterCoatCMO
+  name: Chief Medical Officer's Labcoat
+  description: A Medical Labcoat for the Chief Medical officer. The high collar acts as a splash guard for when a patient's veins are particularly leaky. 
```

Comment:
officer should have a capital, and medical labcoat should be lowercase too. other instances of lab coats have it as two separate words as well